Source/Core/Core/PowerPC/JitArm64/JitArm64_RegCache.cpp

```diff
@@ -176,7 +176,9 @@ Arm64GPRCache::GuestRegInfo Arm64GPRCache::GetGuestCR(size_t preg)
 
 Arm64GPRCache::GuestRegInfo Arm64GPRCache::GetGuestByIndex(size_t index)
 {
-  if (index >= GUEST_GPR_OFFSET && index < GUEST_GPR_OFFSET + GUEST_GPR_COUNT)
+  // We do not need to test for `index >= GUEST_GPR_OFFSET` because
+  // GUEST_GPR_OFFSET is always 0. This otherwise raises a warning.
```

Comment:
I think there should be a `static_assert(GUEST_GPR_OFFSET == 0)` if you're removing the runtime check.